test/performance/visualization/setup-influx-db.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
I guess this setup could be done with influx cli https://docs.influxdata.com/influxdb/v2.7/install/?t=Set+up+with+the+CLI#run-influx-setup-without-user-interaction right?

Reply:
Probably yes, but then everyone also needs to have the client installed. But I'm fine with both ways, as this is just for local development anyways.

Reply:
Wfm, we can always add that later.